source/uwp/Renderer/lib/AdaptiveRenderContext.h

```diff
@@ -31,6 +31,7 @@ namespace AdaptiveNamespace
         IFACEMETHODIMP get_ActionRenderers(_COM_Outptr_ ABI::AdaptiveNamespace::IAdaptiveActionRendererRegistration** value) override;
         IFACEMETHODIMP get_ActionInvoker(_COM_Outptr_ ABI::AdaptiveNamespace::IAdaptiveActionInvoker** value) override;
         IFACEMETHODIMP get_MediaEventInvoker(_COM_Outptr_ ABI::AdaptiveNamespace::IAdaptiveMediaEventInvoker** value) override;
+        IFACEMETHODIMP get_UserInputs(ABI::AdaptiveCards::Rendering::Uwp::IAdaptiveInputs** value) override;
```

Comment:
Please add the _COM_Outptr_ annotation here

Reply:
Done! Thanks